etc/amazon-corretto-crypto-provider-jdk15.security

```diff
@@ -0,0 +1,36 @@
+#
+# List of providers and their preference orders
+
+# Note that we can't just insert ours at the top - we need to restate all
+# providers that are being shifted down to a different index (which is to say,
+# all of them).
+#
+# To ensure this file works on all possible platforms we do include platform specific providers.
+
+security.provider.1=com.amazon.corretto.crypto.provider.AmazonCorrettoCryptoProvider
+security.provider.2=SUN
+security.provider.3=SunRsaSign
+security.provider.4=SunEC
+security.provider.5=SunJSSE
+security.provider.6=SunJCE
+security.provider.7=SunJGSS
+security.provider.8=SunSASL
+security.provider.9=XMLDSig
+security.provider.10=SunPCSC
+security.provider.11=JdkLDAP
+security.provider.12=JdkSASL
+security.provider.13=SunMSCAPI
+security.provider.14=Apple
+security.provider.15=SunPKCS11
+
+#
+# A list of known strong SecureRandom implementations.
+#
+# To help guide applications in selecting a suitable strong
+# java.security.SecureRandom implementation, Java distributions should
+# indicate a list of known strong implementations using the property.
+#
+# This is a comma-separated list of algorithm and/or algorithm:provider
+# entries.
+#
+securerandom.strongAlgorithms=NIST800-90A/AES-CTR-256:AmazonCorrettoCryptoProvider,NativePRNGBlocking:SUN,DRBG:SUN
```

Comment:
i see this is copied over from the other security properties file, but is it still accurate? i believe [we removed the `NIST800-90A/AES-CTR-256` service](https://github.com/corretto/amazon-corretto-crypto-provider/commit/e15bca8ad1cfd57d9a74239683ff1d7b86212a19#diff-36c87648dfaf4bdfe0f2238610f780a9dfef4ad8532944b522a19d7250643c84L82) a few months ago.

Reply:
Oops! Good point. I'll fix that now.